hibernate-core/src/main/java/org/hibernate/cache/internal/EnabledCaching.java

```diff
@@ -95,11 +93,12 @@ public EnabledCaching(SessionFactoryImplementor sessionFactory) {
 					RegionFactory.DEFAULT_QUERY_RESULTS_REGION_UNQUALIFIED_NAME,
 					sessionFactory
 			);
-			regionsByName.put( queryResultsRegion.getName(), queryResultsRegion );
+			domainDataAndQueryResultsRegionNames.add( queryResultsRegion.getName() );
```

Comment:
The default ```QueryResultsRegion``` is not added to ```#legacySecondLevelCacheNames```. Should it be?

If so, should it be qualified?

`
				legacySecondLevelCacheNames.add(
						StringHelper.qualifyConditionally(
								getSessionFactory().getSessionFactoryOptions().getCacheRegionPrefix(),
								queryResultsRegion.getName()
						)
				);
`

Reply:
Practically speaking the default region does have a name, but conceptually that is not the case.  No it should not be added to `legacySecondLevelCacheNames`